project/SparkBuild.scala

```diff
@@ -1059,9 +1059,25 @@ object KubernetesIntegrationTests {
 object DependencyOverrides {
   lazy val guavaVersion = sys.props.get("guava.version").getOrElse("33.1.0-jre")
   lazy val settings = Seq(
-    dependencyOverrides += "com.google.guava" % "guava" % guavaVersion,
-    dependencyOverrides += "jline" % "jline" % "2.14.6",
-    dependencyOverrides += "org.apache.avro" % "avro" % "1.11.3")
+    dependencyOverrides ++= {
+      val nettyVersion = SbtPomKeys.effectivePom.value.getProperties.get(
+        "netty.version").asInstanceOf[String]
+      Seq(
```

Comment:
I personally tend not to add this part of the configuration.

Reply:
Yeah ...

Reply:
Updated.